spec/latest/core/index.html

```diff
@@ -314,6 +314,13 @@ <h3>Adding Affordances to Representations</h3>
       <code>comments</code> property in the following example is intended
       to be dereferenced.</p>
 
+    <p class="note">It is recommended to dereference resources that are
+      within an API's domain. This may prevent possible issues with cross-site
+      scripting or obtaining resources which might have no meaning
+      to the client or the client would be unable to interprete.
+      Still, there is no formal prohibition of dereferencing resources
+      link with well-know properties, e.g. <i>rdf:seeAlso</i>.</p>
```

Comment:
```suggestion
      links with well-known properties, e.g. <i>rdf:seeAlso</i>.</p>
```

Reply:
Actually, I think it should say "linked"